chia/server/ws_connection.py

```diff
@@ -117,6 +117,9 @@ def __init__(
         self.version = ""
         self.protocol_version = ""
 
+    def _get_extra_info(self, name: str) -> Optional[Any]:
+        return self.ws._writer.transport.get_extra_info(name)
```

Comment:
I don't understand how this can help mypy with anything. `get_extra_info()` is part of the standard library, right? I would expect that to be typed already.

Reply:
It's just wrapping so that this assert https://github.com/Chia-Network/chia-blockchain/pull/13878/commits/30d0e1dee21d1154f68e09b27547d54040783747#diff-076ad87bb2203f6728105a8ac4aab4e126ba0113a38e4efff07884ed4934cabfR127 is required once, not thrice. `_writer` is optional and while `ClientWebSocketResponse` contains a `get_extra_info` wrapper, `WebSocketResponse` doesn't but in our case `ws` can be either of both.